### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by meguiraun**
Some fixes, and pollings

### CODE REVIEW

File: MAXIV/BIOMAXBeamInfo.py [hunk 1]
```diff
@@ -41,16 +41,15 @@ def _polling(self):
             except:
                 time.sleep(1)
                 continue
-
             if beam_pos != old_beam_pos:
```
Comment (JieNanMAXIV) [inline MAXIV/BIOMAXBeamInfo.py]: Is this needed!? @meguiraun

Comment (JieNanMAXIV) [inline MAXIV/BIOMAXBeamInfo.py]: Sorry, forget about the above comment. I saw your later commits

File: MAXIV/BIOMAXBeamInfo.py [hunk 2]
```diff
@@ -19,7 +22,53 @@ def init(self):
         self.chan_beam_shape_ellipse = self.addChannel({"type":"exporter", "name":"BeamShapeEllipse"  }, 'BeamShapeEllipse')
         self.chan_ImageZoom=self.addChannel({"type":"exporter", "name":"ImageZoom"  }, 'ImageZoom')
 
+        print "beam x %s and y %s " % (self.chan_beam_pos_x.getValue(),self.chan_beam_pos_y.getValue())
```
Comment (JieNanMAXIV) [inline MAXIV/BIOMAXBeamInfo.py]: chan_beam_pos_x chan_beam_pos_y are not defined

Comment (meguiraun) [inline MAXIV/BIOMAXBeamInfo.py]: ??? see lines 18 ad 19... expand the line-viewer?

File: MAXIV/BIOMAXBeamInfo.py [hunk 3]
```diff
@@ -49,11 +98,11 @@ def get_beam_position(self):
         Return    :
         """
         logging.info ("camera is %s", str(self.camera))
-        try:
+	if self.chan_ImageZoom.getValue() is not None:
```
Comment (JieNanMAXIV) [inline MAXIV/BIOMAXBeamInfo.py]: when chan_ImageZoom is None, it will give an error with "if"

File: MicrodiffMotor.py [hunk 4]
```diff
@@ -128,15 +129,15 @@ def getDynamicLimits(self):
...
-    def motorPositionChanged(self, absolutePosition, private={}):
-        if None in (absolutePosition, self.old_position):
-          return
-        if abs(absolutePosition - self.old_position)<=1E-3:
-          return 
+
+    def motorPositionChanged(self, absolute_position, private={}):
+        if None in (absolute_position, self.old_position):
+            return
+        if abs(absolute_position - self.old_position) <= self.motor_resolution:
```
Comment (JieNanMAXIV) [inline MicrodiffMotor.py]: motor_resolution is not defined.

**Pull request merged**